### ISSUE TITLE
Evaluation results on the uncompressed dataset.

### ISSUE THREAD

**Issue by qtct** [researchmm/FTVSR#24]
Hello, I tried to use your model to train on the general ×4 video super-resolution task. Vimeo90K is used for the training set, and the degraded model is BD. Vid4 is used for the validation set, and the degraded model is BD. The effect seems not to be very good, which is worse than BasicVSR. Is that true?
